test/decl/func/operator.swift

```diff
@@ -147,7 +147,7 @@ postfix prefix func ++(x: Int) {} // expected-error {{attribute 'postfix' cannot
 
 // Don't allow one to define a postfix '!'; it's built into the
 // language.
-postfix operator!  // expected-error {{cannot declare a custom postfix '!' operator}}
+postfix operator!  // expected-error {{cannot declare a custom postfix '!' operator}} expected-error {{expected operator name in operator declaration}}
```

Comment:
tiny nitpick: should there be a space in `operator !`, like there is for `operator ?` on the next line?